guide/ops/catalog/index.md

```diff
@@ -120,14 +120,14 @@ brooklyn.catalog:
   brooklyn.libraries:
   - http://some.server.or.other/path/my.jar
   items:
-  - classpath://my-catalog-entries-inside-jar.bom
+  - classpath://myJar-BundleSymbolicName:my-catalog-entries-inside-jar.bom
   - some-property: value
-    include: classpath://more-catalog-entries-inside-jar.bom
+    include: classpath://myJar-BundleSymbolicName:more-catalog-entries-inside-jar.bom
   - id: use-from-my-catalog
     version: "1.0.0"
     itemType: entity
     item:
-      type: some-type-defined-in-my-catalog-entries
+      type: myJar-BundleSymbolicName:some-type-defined-in-my-catalog-entries
```

Comment:
Same as previous comment, prefixes shouldn't be required. Actually should be discouraged and used as a last resort to get things working.
File bugs against Brooklyn if you find cases where they are needed.